tedana/utils.py

```diff
@@ -250,10 +250,10 @@ def threshold_map(img, min_cluster_size, threshold=None, mask=None,
         Boolean array for masking resultant data array. Default is None.
     binarize : bool, optional
         Default is True.
-    sided : {'two', 'one', 'bi'}, optional
+    sided : {'bi', 'two', 'one'}, optional
         How to apply thresholding. One-sided thresholds on the positive side.
         Two-sided thresholds positive and negative values together. Bi-sided
-        thresholds positive and negative values separately. Default is 'two'.
+        thresholds positive and negative values separately. Default is 'bi'.
     """
```

Comment:
Should this section specify the details of what is returned?

Reply:
Added.